controllers/healthcheck_controller.go

```diff
@@ -90,13 +90,15 @@ func (r *HealthCheckReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error)
 	ctx := context.Background()
 	log := r.Log.WithValues(hcKind, req.NamespacedName)
 
+	log.Info("Starting HealthCheck reconcile ...", "for this", "hc")
```

Comment:
Can we add some specifics to this log message (e.g. which HealthCheck object) and reformat it?

Reply:
Sure made it more readable the log will now be like:
2020-01-30T07:54:10.122+0530    INFO    controllers.HealthCheck Starting HealthCheck reconcile for ...  {"HealthCheck": "health/dns-healthcheck-qhmsk"}